[EDU-5295] drizzle with turso template guide (#1333)
aziontech/docs: src/content/docs/pt-br/pages/guias/marketplace/templates/turso-with-drizzle.mdx

---

> title: Como implantar o template Turso Starter Kit with Drizzle

Reviewer: ```suggestion
title: Como implantar o template Turso Starter Kit with Drizzle ORM
```

> description: Este template contém as configurações para conectar uma Edge Application a um banco de dados Turso.

Reviewer: ```suggestion
description: Este template contém as configurações para conectar uma edge application a um banco de dados Turso.
```

meta_tags: Drizzle ORM, Turso database, Edge Function, Azion Marketplace, web, guides, templates
namespace: docs_guides_turso_with_drizzle
permalink: /documentacao/produtos/guias/turso-with-drizzle/
---

import Tag from 'primevue/tag';

<Tag severity="info" client:only="vue">
Preview
</Tag>

O template **Turso Starter Kit with Drizzle ORM** contém as configurações para criar e conectar uma edge application a um banco de dados Turso via Drizzle ORM.

A implantação deste template cria um repositório do GitHub contendo seu projeto, além de uma edge application e um domínio para facilitar seu acesso e gerenciamento através da Plataforma de Edge da Azion. Além disso, cria um novo banco de dados com uma nova tabela na sua conta Turso.

---

## Requisitos

- Uma [conta GitHub](https://github.com/signup) para se conectar com a Azion e criar seu novo repositório.
    - Cada push será implantado automaticamente nesse repositório para manter seu projeto atualizado.
- Uma [conta Turso](https://turso.tech/) com um slot de banco de dados vazio.

> - Este template usa [Application Accelerator](/pt-br/documentacao/produtos/build/edge-application/application-accelerator/), [Edge Functions](/pt-br/documentacao/produtos/build/edge-application/edge-functions/), e [Edge Cache](/pt-br/documentacao/produtos/build/edge-application/edge-cache/). Isso pode gerar custos relacionados ao uso. Consulte a [página de preços](/pt-br/documentacao/produtos/precos/) para mais informações.
> 
> :::note
> O armazenamento e as operações do banco de dados também podem gerar custos diretamente pelo Turso. Verifique a [página de preços do Turso](https://turso.tech/pricing) para obter detalhes.
> :::

Reviewer: ```suggestion

Este template usa [Application Accelerator](/pt-br/documentacao/produtos/build/edge-application/application-accelerator/), [Edge Functions](/pt-br/documentacao/produtos/build/edge-application/edge-functions/), e [Edge Cache](/pt-br/documentacao/produtos/build/edge-application/edge-cache/). Isso pode gerar custos relacionados ao uso. Consulte a [página de preços](/pt-br/documentacao/produtos/precos/) para mais informações. O armazenamento e as operações do banco de dados também podem gerar custos diretamente na Turso. Verifique a [página de preços da Turso](https://turso.tech/pricing) para obter detalhes.
```

---

## Implante o template

Você pode obter e configurar seu template através do Azion Console. Para implantá-lo facilmente no edge, clique no botão abaixo.

<LinkButton

>     label="Deploy"

Reviewer: ```suggestion
    label="Implantar"
```
